CHANGELOG.md

```diff
@@ -78,6 +78,7 @@ The types of changes are:
 * Frontend - Configure a Manual entry connector [#1015](https://github.com/ethyca/fidesops/pull/1247)
 * Manual webhook test functionality [#1323](https://github.com/ethyca/fidesops/pull/1323/)
 * Added change request table [#1340](https://github.com/ethyca/fidesops/pull/1340)
+* Adds Access Endpoints for Braze [#1248](https://github.com/ethyca/fidesops/pull/1248)
```

Comment:
Changed to `Access and erasure support for Braze` and moved it to the `Unreleased` section.